### PR TITLE
support to load groovy functions from specified path

### PR DESCRIPTION
**Pull request by vahurkrouverk**
Added loading of groovy functions for DEL from specified paths (file path or url).

### CODE REVIEW

File: dialob-components/dialob-groovy-functions/src/main/java/io/dialob/groovy/DialobGroovyFunctionsAutoConfiguration.java [hunk 1]
```diff
@@ -27,10 +29,11 @@
 public class DialobGroovyFunctionsAutoConfiguration {
 
   @Bean
-  public GroovyFunctionRegistry groovyFunctionRegistry(ApplicationContext applicationContext, FunctionRegistry functionRegistry) {
+  public GroovyFunctionRegistry groovyFunctionRegistry(ApplicationContext applicationContext, FunctionRegistry functionRegistry, DialobSettings dialobSettings) {
     GroovyFunctionRegistry groovyFunctionRegistry = new GroovyFunctionRegistry(applicationContext, functionRegistry);
     List<String> functions = new ArrayList<>();
     functions.add("classpath:/scripts/FoaasService.groovy");
```
Comment (patope) [inline dialob-components/dialob-groovy-functions/src/main/java/io/dialob/groovy/DialobGroovyFunctionsAutoConfiguration.java:35]: This have been an example. Remove FoaasService...

File: dialob-config-parent/dialob-settings/src/main/java/io/dialob/settings/FunctionSettings.java [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+import lombok.Data;
+
+@Data
+public class FunctionSettings {
+  
+  private GroovyFunctionSettings groovy = new GroovyFunctionSettings();
+  
+  @Data
+  public static class GroovyFunctionSettings {
+    private List<String> path = new ArrayList<>();
```
Comment (patope) [inline dialob-config-parent/dialob-settings/src/main/java/io/dialob/settings/FunctionSettings.java:30]: rename as `locations`

File: dialob-components/dialob-groovy-functions/src/main/java/io/dialob/groovy/DialobGroovyFunctionsAutoConfiguration.java (hunk 1 above)
Comment (vahurkrouverk) [reply dialob-components/dialob-groovy-functions/src/main/java/io/dialob/groovy/DialobGroovyFunctionsAutoConfiguration.java]: This example function removed

File: dialob-config-parent/dialob-settings/src/main/java/io/dialob/settings/FunctionSettings.java (hunk 2 above)
Comment (vahurkrouverk) [reply dialob-config-parent/dialob-settings/src/main/java/io/dialob/settings/FunctionSettings.java]: renamed attribute to `locations`

**Review by patope — APPROVED** [review]

**Pull request merged**